src/engine/libengine/MethodPhonetic.cpp

```diff
@@ -718,7 +714,7 @@ IMCommand MethodPhonetic::handleSpecialKey(int key) {
   }
 
   // Control flow should not reach here.
-  return ret;
+  //return ret;
```

Comment:
Same here...

Reply:
Here is a sample  example

![image](https://user-images.githubusercontent.com/4653623/43994494-a1b86490-9dbf-11e8-850a-e3e23a516c99.png)